### ISSUE TITLE
Cloud Height draggable button needs better indication

### ISSUE THREAD

**Issue by catthou** [Ficklampan/TinyMod#13]
The Cloud Height button seems like something you should be clicking. There needs to be some indication that it's meant to be dragged. A simple tooltip would suffice.

**Comment by EdvinVikstrom** [comment]
yes i know maybe gonna replace it with a text field(with the draggable function)
